mobify/astro-scaffold
# Android only supports handsets

Google play will only allow app to be installed on handsets

JIRA: None
Linked PRs: https://github.com/mobify/thinkgeek-ios/pull/422
https://github.com/mobify/ritchie-brothers-app/blob/develop/android/RitchieBrothers/src/main/AndroidManifest.xml#L13

## Changes
- Specify supported devices in `AndroidManifest`

## How to test-drive this PR
- Unfortunately the only way to test is to publish in the app store
- Please see linked pr/code from thinkgeek and ritchie which have both published to the app store using this method

## TODOS:
- [x] Change works in both Android
